cita-network/src/config.rs

```diff
@@ -37,32 +42,54 @@ impl NetConfig {
     }
 }
 
+#[derive(Debug, Deserialize, Clone)]
+pub struct AddressConfig {
+    pub addr: Address,
+}
+
+impl AddressConfig {
+    pub fn new(path: &str) -> Self {
+        let mut buffer = String::new();
+        File::open(path)
+            .and_then(|mut f| f.read_to_string(&mut buffer))
+            .unwrap_or_else(|err| panic!("[Config] Error while loading node address: [{}]", err));
+
+        let addr = Address::from_str(clean_0x(&buffer)).unwrap();
+
+        AddressConfig { addr }
+    }
+}
+
 #[cfg(test)]
 mod tests {
     use super::NetConfig;
     use std::io::Write;
     use tempfile::NamedTempFile;
+
     #[test]
-    fn basics() {
+    fn basic_test() {
         let toml_str = r#"
-        port = 40000
+        port = 4000
         enable_tls = true
+        max_connects = 4
+        id_card = 9
         [[peers]]
-        ip = "127.0.0.1"
-        port = 40001
-        common_name = "test1.cita"
+            ip = "0.0.0.0"
```

Comment:
It is weird. Why?

Reply:
Oh, you are saying the value of `ip` or the indentation of the code?

Reply:
I had a chat with @jerry-yu about it at lunch time, we both think the IP should not be `"0.0.0.0"`.

Reply:
Got it!